x/poe/keeper/contracts.go

```diff
@@ -51,7 +51,7 @@ func (k Keeper) StakeContract(ctx sdk.Context) StakeContract {
 }
 
 type EngagementContract interface {
-	UpdateAdmin(ctx sdk.Context, newAdmin, sender sdk.AccAddress) error
+	UpdateBootstrapAccountAddress(ctx sdk.Context, newAdmin, sender sdk.AccAddress) error
```

Comment:
As above, please revert the renaming for this interface and other implementations